akka-docs/src/main/paradox/typed/replicated-eventsourcing.md

```diff
@@ -237,48 +237,17 @@ When comparing two version vectors `v1` and `v2`:
 * `v1`is AFTER `v2` iff for all i `v1(i) >= v2(i)` and there exist a j such that `v1(j) > v2(j)`
 * `v1`is CONCURRENT with `v2` otherwise
 
-## Tagging events and running projections
+## Running projections
 
-Just like for regular `EventSourcedBehavior`s it is possible to tag events along with persisting them. 
-This is useful for later retrival of events for a given tag. The same @ref[API for tagging provided for EventSourcedBehavior](persistence.md#tagging) can 
-be used for replicated event sourced behaviors as well.
-Tagging is useful in practice to build queries that lead to other data representations or aggregations of these event 
-streams that can more directly serve user queries – known as building the “read side” in @ref[CQRS](cqrs.md) based applications.
+Just like for regular `EventSourcedBehavior`s it is possible to create read side projections with @extref[Akka Projections](akka-projection:).
 
-Creating read side projections is possible through @extref[Akka Projections](akka-projection:)
-or through direct usage of the @ref[events by tag queries](../persistence-query.md#eventsbytag-and-currenteventsbytag).  
+When running a Projection in each replica you should be aware of that it will receive all events, from all replicas.
+Since the events can arrive in different order, they can also come in different order per replica.
 
-The tagging is invoked in each replicas, which requires some special care in using tags, or else the same event will be
```

Comment:
I had a hard time understanding what we were trying to say here. Anyway, demphasized tagging since slices is the recommended way.